build_projects/dotnet-host-build/PrepareTargets.cs

```diff
@@ -42,6 +42,17 @@ public class PrepareTargets
         public static BuildTargetResult Init(BuildTargetContext c)
         {
             var configEnv = Environment.GetEnvironmentVariable("CONFIGURATION");
+            string platformEnv = Environment.GetEnvironmentVariable("TARGETPLATFORM") ?? RuntimeEnvironment.OperatingSystemPlatform.ToString();
+            string targetRID = Environment.GetEnvironmentVariable("TARGETRID");
+            if (targetRID == null)
+            {
+                targetRID = RuntimeEnvironment.GetRuntimeIdentifier();
+                if (targetRID.StartsWith("win") && (targetRID.EndsWith("x86") || targetRID.EndsWith("x64")))
+                {
+                    targetRID = $"win7-{RuntimeEnvironment.RuntimeArchitecture}";
+                }
+            }
+            string targetFramework = Environment.GetEnvironmentVariable("TARGETFRAMEWORK") ?? "netcoreapp1.0";
```

Comment:
Configuration already set the precedent by being passed via environment variables so I followed the scheme already in place. It would take a fairly large amount of refactoring to change it to use Main args since there are some static initiatializations (e.g. Dirs.cs) that also check these environment variables. 

Since I imagine this whole repo will at some point be rewritten to use msbuild like CLI is now doing, it's likely not worth changing now.